.changeset/ten-meals-count.md

```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/frontify-cli": patch
```

Comment:
you can move that into the four-beers-smell, no need to have an own changeset file